src/hotspot/os/aix/globals_aix.hpp

```diff
@@ -49,11 +49,12 @@
           "Allow VM to run with EXTSHM=ON.")                                        \
                                                                                     \
   /*  Maximum expected size of the data segment. That correlates with the      */   \
-  /*  to the maximum C Heap consumption we expect.                             */   \
-  /*  We need to know this because we need to leave "breathing space" for the  */   \
-  /*  data segment when placing the java heap. If that space is too small, we  */   \
-  /*  reduce our chance of getting a low heap address (needed for compressed   */   \
-  /*  Oops).                                                                   */   \
+  /*  maximum C Heap consumption we expect.                                    */   \
+  /*  We need to leave "breathing space" for the data segment when             */   \
+  /*  placing the java heap. If the MaxExpectedDataSegmentSize setting         */   \
+  /*  is too small, we might run into resource issues creating many native     */   \
+  /*  threads, if it is too large, we reduce our chance of getting a low hea   */   \
```

Comment:
typo: hea -> heap

Reply:
Thanks for correcting the comment; it made no sense before